simple/msg_pingpong.c

```diff
@@ -40,6 +44,8 @@
 #include <rdma/fi_cm.h>
 #include <shared.h>
 
+static uint32_t FI_VER = FI_VERSION(1, 0);
```

Comment:
Yes that sounds good. How about I remove this for now and put it in a separate commit, I wanted to keep these changes to msg_pingpong only.